config/routes.rb

```diff
@@ -1,25 +1,63 @@
-Rails.application.routes.draw do
+Rails.application.routes.draw do  
+  # PART 1: EACH IN ERB
```

Comment:
I'm not sure I understand what the instruction title means. It seems like they need to make the RCAV for `/lottery/lucky` and `/lottery/unlucky` which is different than just editing an ERB file.

Reply:
The new concept here is `.each` within ERB; I plan to demo this with them. Initially I had one example one stubbed out (lucky) and then was going to have them do unlucky. Then I deleted it so I could live type it in front of them. I may put it back again.

Reply:
Okay! I think the README needs to be updated then, since it mentions that those two Routes are done already.